languagetool-core/src/test/java/org/languagetool/rules/patterns/PatternRuleTest.java

```diff
@@ -90,24 +89,23 @@ public void testSupportsLanguage() {
     assertFalse(patternRuleVariant1.supportsLanguage(fakeLanguage1WithVariant2));
   }
   
-  @Ignore
   @Test
   public void shortMessageIsSmallerThanErrorMessage() throws IOException {
     for (Language lang : Languages.get()) {
       MultiThreadedJLanguageTool languageTool = new MultiThreadedJLanguageTool(lang);
       for (AbstractPatternRule rule : getAllPatternRules(lang, languageTool)) {
-        failIfShortMessageSmallerThanErrorMessage(rule);
+        warnIfShortMessageSmallerThanErrorMessage(rule);
       }
     }
   }
 
-  private void failIfShortMessageSmallerThanErrorMessage(AbstractPatternRule rule) {
+  private void warnIfShortMessageSmallerThanErrorMessage(AbstractPatternRule rule) {
```

Comment:
Yes, sometimes it's hard to come up with good method names...

On this case the we're using the >= operator:
sizeOfShortMessage >= sizeOfErrorMessage
So if we'd like to be very precise we should probably name it something like
warnIfShortMessageLongerThanOrEqualInSzeToErrorMessage
but that would be a little too verbose ;-)